crates/typed-store/src/traits.rs

```diff
@@ -130,6 +138,14 @@ where
     fn try_catch_up_with_primary(&self) -> Result<(), Self::Error>;
 }
 
+pub enum IterRangeBound<K>
```

Comment:
I think you can replace this with `RangeBounds`? 
std::ops::RangeBounds

Reply:
+1, this looks very similar to `RangeBounds`

Reply:
Interestingly the shorthand range syntax does not have a way to exclude the starting key.
But I've changed the code to use `std::ops:RangeBounds`, thanks for the pointer